src/api_server.rs

```diff
@@ -85,11 +84,11 @@ impl ApiServer {
                     Some(config_json) => config_json,
                     None => return Err(anyhow!("The config of virtio-net device is required")),
                 };
-                let configs: Vec<VirtioNetDeviceConfigInfo> = serde_json::from_str(config_json)
-                    .context("Parse virtio-net device config from json")?;
+                let configs: Vec<NetworkInterfaceConfig> = serde_json::from_str(config_json)
+                    .context("Parse NetworkInterfaceConfig from JSON")?;
                 for config in configs.iter() {
                     self.insert_virnet(config.clone())
-                        .context("Insert a virtio-net device to the Dragonball")?;
+                        .context("Insert a virtio device to the Dragonball")?;
```

Comment:
The user cannot know that it's net device that went wrong in this error message.

Reply:
Fixed.